com.woltlab.wcf/option.xml

```diff
@@ -1615,6 +1615,11 @@ DESC:wcf.global.sortOrder.descending</selectoptions>
 				<optiontype>boolean</optiontype>
 				<defaultvalue>0</defaultvalue>
 			</option>
+			<option name="enable_deprecation_warnings">
```

Comment:
You should probably set a dependency on the dev tool option here.

Reply:
I have done the same for `log_missing_language_items`.